packages/SystemsThinkingWorkbench-Core.package/STWConnection.class/instance/addMiddleHandlesTo..st

```diff
@@ -0,0 +1,27 @@
+halos and balloon help
+addMiddleHandlesTo: aHalo
+
+	| synchonicity |
+	
+	self isSynchronous 
+		ifTrue: [synchonicity := 's' asMorph] 
+		ifFalse: [synchonicity := 'a' asMorph].
+	aHalo
+		addMiddleHandleFor: [self delete]
+		color: Color red muchLighter
+		containing: (ScriptingSystem formAtKey: #'Halo-Dismiss') asMorph;
+		
+		addMiddleHandleFor:
+			[self isSynchronous
+				ifTrue:
+					[self beAsynchronous.
+					synchonicity contents: 'a'.]
```

Comment:
Isn't it more reasonable to use an instance variable for synchronicity and two constances (one for `'a'` and one for `'s'`) so synchronicity could be set in `beSynchronous` and in `beAsynchronous`?